test/mp_correct/src/do20.f90

```diff
@@ -0,0 +1,27 @@
+!* Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+!* See https://llvm.org/LICENSE.txt for license information.
+!* SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+!	parallel do with negative stride
+
+program main
+
+  call mmm(10, 1, -2)
+
+contains
+
+subroutine mmm (m1, m2, m3)
+
+    integer :: m, m1, m2, m3
+
+
```

Comment:
Can you delete unnecessary blank lines and make your indentation consistent (2 spaces per level)?